pilot/pkg/serviceregistry/external/servicediscovery_test.go

```diff
@@ -43,18 +44,27 @@ func createServiceEntries(serviceEntries []*networking.ServiceEntry, store model
 	}
 }
 
-func initConfigStore() model.IstioConfigStore {
-	configDescriptor := model.ConfigDescriptor{
-		model.ServiceEntry,
-	}
-	store := memory.Make(configDescriptor)
+type channelTerminal struct {
```

Comment:
What the stopfn and the channel needed for?

Reply:
It's just for completeness ... the config controller is now being run and stopped.